### PR TITLE
Added a case studies details page

### PR DESCRIPTION
**Pull request by DevTaehong**
- Add a case studies details page

1. Hero
2. Teck Stack
3. Problem statement
4. Figma Design section
5. My process
6. Challenges & Learnings
7. Other case studies

NOTE: I've made components for svg icons of case studies for colour changing 
(Definition.tsx, Deployment.tsx, Design.tsx, Development.tsx, Planning.tsx) 

**Desktop**

https://github.com/DevTaehong/portfolio_jsm/assets/71358207/47ea7aa7-b31d-45b3-a38c-6cee29f93ec3

**Mobile**

https://github.com/DevTaehong/portfolio_jsm/assets/71358207/3c8d9a44-6385-4423-baf3-5d094bff871a

### CODE REVIEW

File: components/caseStudyDetails/ChallengeLearning.tsx [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+  challengesOrLearnings: string;
+  bulletPoints: string[];
+  icon: string;
+}) => {
+  return (
+    <Card className="challengesLearnings border-0 shadow-none dark:bg-black300 sm:gap-[1.88rem] sm:px-10 sm:py-9">
+      <h6
+        className={`bodyBold ${
+          challengesOrLearnings === "CHALLENGES"
+            ? "text-[#E15A46]"
```
Comment (Nassim-Saboundji) [inline components/caseStudyDetails/ChallengeLearning.tsx:18]: Please don't hardcode the colors in your code, put them in your tailwind config.

**Review by Nassim-Saboundji — CHANGES_REQUESTED** [review]
The videos you shared in the PR description don't work. Can you fix this?

**Comment by DevTaehong** [comment]
> The videos you shared in the PR description don't work. Can you fix this?

Yes, I can.
I've made them GIFs.

268426949-ca153803-d734-48e9-9772-d778a0b6e581
268427040-d1db01a2-f7f1-4bc8-bb30-91c912981f53

File: components/caseStudyDetails/ChallengeLearning.tsx (hunk 1 above)
Comment (DevTaehong) [reply components/caseStudyDetails/ChallengeLearning.tsx]: Hi @Nassim-Saboundji, I put them in my tailwind config.

**Pull request merged**